warehouse/config.py

```diff
@@ -238,6 +239,13 @@ def configure(settings=None):
         coercer=int,
         default=300,
     )
+    maybe_set(
+        settings,
+        "token.remember_device.max_age",
+        "TOKEN_REMEMBER_DEVICE_MAX_AGE",
+        coercer=int,
+        default=2592000,  # 30 days,
```

Comment:
I've started to want to migrate "magic seconds" to more readable values.

```python
from datetime import timedelta
...
        default=timedelta(days=30).total_seconds(),
...
```

Reply:
Good idea!